convert2rhel/unit_tests/subscription_test.py

```diff
@@ -1519,6 +1519,30 @@ def test_update_rhsm_custom_facts_no_rhsm(global_tool_opts, caplog, monkeypatch)
     assert "Skipping updating RHSM custom facts." in caplog.records[-1].message
 
 
+@pytest.mark.parametrize(
+    ("telemetry_disabled", "rhsm_called", "message"),
+    [
+        (True, 0, "Skipping updating RHSM custom facts. Telemetry disabled."),
+        (False, 1, "RHSM custom facts uploaded successfully."),
```

Comment:
Not sure that we need this parametrization here. We already have a test that covers that the upload was successful. The `test_update_rhsm_custom_facts`.

I would only focus on the case where the telemetry is disabled here, as the test name is mostly related to that.